test/test_ssl.py

```diff
@@ -63,7 +63,7 @@ def test_context_sni_with_ip_address(monkeypatch, has_sni, server_hostname, uses
     if uses_sni:
         context.wrap_socket.assert_called_with(sock, server_hostname=server_hostname)
     else:
-        context.wrap_socket.assert_called_with(sock)
+        context.wrap_socket.assert_called_with(sock, server_hostname=None)
```

Comment:
I think this will hurt Python versions that have `ssl` but don't have SNI support?
Might be good to verify this.

Reply:
The default parameter for ``server_hostname`` is actually None.  It made the code I needed to add simpler to add this as a parameter but I'll double-check it doesn't impact Python versions without SNI.

Reply:
Thank you! Will leave this conversation open until that's verified.